### PR TITLE
Provide a mechanism for developers to test kickstart ERBs changes

### LINKED ISSUE #97: Need a mechanism to test updates to kickstarts and kickstart partials
Currently there is no easy way for developers to test updates to the kickstart.ERB or the
kickstart partials without running a full build.

A simple mechanism is needed to allow a developer to ensure the generated kickstarts
are correct when testing changes to the kickstart.ERBs and partials.

### PR DESCRIPTION
**Pull request by jvlcek**
Fixes #97

### CODE REVIEW

**Comment by jvlcek** [comment]
This PR is only for a development tool/script that will allow developers to test kickstart ERB changes directly from a manageiq-appliance-build checkout.

To test changes to a kickstart ERB a developer would clone their fork of the manageiq-appliance-build repository, make the proposed changes to the kickstart ERB and/or partials, run the script proposed by this PR and manually evaluate the output generated in <manageiq-appliance-build repo directory>/kickstarts/generated

e.g.:
`
clone https://github.com/my/manageiq-appliance-build.git
cd manageiq-appliance-build
vim kickstarts/base.ks.erb # making proposed updates
cd scripts
bundle exec ruby kickstart_generator.rb
cat ../kickstarts/generated/*
 `

**Comment by jvlcek** [comment]
@carbonin Please review.

**Comment by carbonin** [comment]
I think this might be better somewhere other than the source file itself.
Maybe make it a rake task or a ruby script in the `bin` directory.

What do you think @abellotti @jrafanie ?

**Comment by jvlcek** [comment]
@carbonin Thank you for the reply. Can you please elaborate on "why" you think moving this out of the source file would be an improvement.

**Comment by abellotti** [comment]
Yeah, let's go with a seperate script, that would allow running it from anywhere. How about bin/kickstart_build.sh

**Comment by carbonin** [comment]
@jvlcek I think that leaving that in the same file doesn't distinguish clearly enough that it is for dev/test purposes rather than a step in the actual build process.  Plus moving it out removes the filename hack that you had to put in to not let it run when being required.

**Comment by jvlcek** [comment]
@carbonin OK I can see that.
@abellotti Thank you for the input.
I'll move it to it's own script in bin/kickstart_build.rb (.rb not .sh)

Thanks!

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,6 @@
 /pkg
 /productization
+kickstarts/generated/base.ks
+kickstarts/generated/*.json
+kickstarts/generated/*.cfg
```
Comment (carbonin) [inline .gitignore]: Should probably also add `scripts/Gemfile.dev.rb`

Comment (jvlcek) [inline .gitignore]: Good point @carbonin! I had missed it because I have `Gemfile.dev.rb` in my $HOME/.gitignore
:+1:

**Comment by carbonin** [comment]
LGTM :+1:

**Comment by kbrock** [comment]
:+1: nice job @jvlcek

**Comment by jrafanie** [comment]
Nice @jvlcek @carbonin @abellotti, looks good.

**Pull request merged**